### PR TITLE
Updates to shared mini card

### PR DESCRIPTION
**Pull request by amponce**
**Note**: 

This change updates the behavior of the mini card to allow for reuse in other components. There are a few existing components that will need to update the `subhead` and `onDelete` functionality to `body` and pass `showDelete` = true 

## Summary

-  Updates mini card component to support removing delete and adds body as a more general param for additional content

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#NA

## Testing done

-  Delete was required 
-  Subhead was a param before being changed to body

## Screenshots
_Note: This field is mandatory for component work and UI changes (non-component work should NOT have screenshots)._

| | Mockup | 
| --- | --- |
| Desktop |Screenshot 2023-03-08 at 12 25 57 PM | 

| | Card with delete | 
|--|--|
| Desktop | image |
| Mobile | image |

## What areas of the site does it impact?
Enhanced FSR

## Acceptance criteria

- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature
- [ ]  [Accessibility foundational testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### CODE REVIEW

**Review by OptionSelect — APPROVED** [review]
LGTM! 🚀

File: src/applications/financial-status-report/components/shared/MiniSummaryCard.jsx [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        </div>
+        <div className="medium-screen:vads-l-row small-desktop-screen:vads-l-col--3 summary-card-button-container">
+          <Link
+            aria-label={`Edit ${heading}`}
+            to={editDesination}
+            className="vads-u-padding--1"
+          >
+            <span className="vads-u-font-size--h3">
+              Edit
+              <i
```
[the remark this answers is not in the document]
Comment (Dr-Pongo) [reply src/applications/financial-status-report/components/shared/MiniSummaryCard.jsx]: Icons have `aria-hidden="true"`

File: src/applications/financial-status-report/components/shared/MiniSummaryCard.jsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            </span>
+          </Link>
+          {showDelete && (
+            <button
+              type="button"
+              aria-label={`Delete ${heading}`}
+              className="usa-button summary-card-delete-button vads-u-margin--1"
+              onClick={onDelete}
+            >
+              <i
```
[the remark this answers is not in the document]
Comment (Dr-Pongo) [reply src/applications/financial-status-report/components/shared/MiniSummaryCard.jsx]: Icons have `aria-hidden="true"`

File: src/applications/financial-status-report/components/shared/MiniSummaryCard.jsx [hunk 3]
```diff
@@ -6,41 +6,43 @@ import { Link } from 'react-router';
...
+      </div>
+      <div className="vads-l-row vads-u-justify-content--space-between vads-u-align-items--center">
+        <Link
+          aria-label={`Edit ${heading}`}
+          to={editDesination}
+          className="vads-u-padding-y--1 vads-u-padding-x--2"
+        >
+          <span className="vads-u-font-size--h3">
+            <strong>Edit</strong>
+            <i
```
[the remark this answers is not in the document]
Comment (Dr-Pongo) [reply src/applications/financial-status-report/components/shared/MiniSummaryCard.jsx:38]: Icons have `aria-hidden="true"`

**Pull request merged**